deploy/cluster-monitoring-config-non-uwm-staging-integration/cluster-monitoring-config.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    prometheusK8s:
+    # If the location of this config changes,
+    # make sure to adjust https://github.com/openshift/ops-sop/blob/master/v4/howto/o11m_mst_killswitch.md
+    # so that killswitch instructions point to the right file!
+      remoteWrite:
+        - url: http://token-refresher.openshift-monitoring.svc.cluster.local
```

Comment:
I just remembered that we don't want to rollout remote writes to 4.5 clusters which this file targets.

Reply:
I believe the `non-uwm` configuration should be the same for both production and non-production clusters. So they can probably be merged.